src/main/fc/rc_adjustments.c

```diff
@@ -664,6 +669,33 @@ static uint8_t applySelectAdjustment(adjustmentFunction_e adjustmentFunction, ui
     return position;
 }
 
+static void calcActiveAdjustmentRanges(void)
+{
+    adjustmentRange_t defaultAdjustmentRange;
+    memset(&defaultAdjustmentRange, 0, sizeof(defaultAdjustmentRange));
+
+    activeAdjustmentRangeCount = 0;
+    for (int i = 0; i < MAX_ADJUSTMENT_RANGE_COUNT; i++) {
+        const adjustmentRange_t * const adjustmentRange = adjustmentRanges(i);
+        if (memcmp(adjustmentRange, &defaultAdjustmentRange, sizeof(defaultAdjustmentRange)) != 0) {
```

Comment:
Isn't there some function to test if range is valid/enabled?

Reply:
I couldn't find anything. The processing has a variety of individual tests for the "validity" of the entry but I'm really just looking to see if it has any of the elements configured to avoid replicating the validation logic.  When the adjustment is disabled in the configurator the values of all elements are set to 0.